### DISCUSSION TITLE
안녕하세요? IWP구매 관련해서 문의 입니다 :)

### DISCUSSION THREAD

**Discussion by changhyeok-an** [naverz/zepeto-studio-kor#845] [5. Q&A]
안녕하세요? 제페토 IWP구매쪽이 막혀가지고 이렇게 질문을 남깁니다.
현재 제페토 purchaseCallback와 코드 추가 그리고 실험삼아서 유니티에 아이템을 추가했습니다.
잘 작동은 하는데 계속 이런 모양이 떠서 어떻게 해야 할지 모르겠습니다.

<img width="970" alt="스크린샷 2022-06-06 오후 4 49 46" src="https://user-images.githubusercontent.com/83633279/172118820-efc102e1-ff91-478e-8fe9-556dfeead871.png">

제페토에 나와있는 설명처럼 IWP는 다 추가한 상태입니다

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 등록한 월드에 커버 이미지(썸네일)이 없을 경우 위와 같은 메세지가 뜰 수 있습니다.
확인 부탁 드립니다.
